### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by rolznz** [getAlby/bitcoin-connect#182]
Tabbing / Keyboard access should work in all places. Some elements (like the button below) should be focused by default.

![image](https://github.com/getAlby/bitcoin-connect/assets/33993199/5a17987e-2eb6-442f-a749-d7225eb0396a)

**Comment by harith-hacky03** [comment]
Can you briefly describe the problem? I am ready to fix it

**Comment by rolznz** [comment]
Hi @harith-hacky03 this is a general task about keyboard accessibility throughout the library. I don't believe we can easily use Bitcoin Connect without a mouse right now for example. Some good information is here: https://usability.yale.edu/web-accessibility/articles/focus-keyboard-operability

**Comment by itstomekk** [comment]
Small reminder: there is a **bounty** for solving this issue!
I have posted it across multiple bounty platforms today, maybe someone will pick it up!

[**Bitcoin Connect**](https://bitcoin-connect.com) is an embeddable, open-source JavaScript library maintained by [Alby](https://getalby.com) that allows websites to easily integrate Bitcoin and Lightning functionalities. It provides a modal widget for user interactions, enabling seamless onboarding and transactions.

Currently, the Bitcoin Connect modal widget requires a mouse for navigation and interaction, limiting its accessibility. By improving keyboard accessibility, we aim to make the library more inclusive and user-friendly for all users, including those with disabilities who rely on keyboard navigation.

## 💰 Bounty: 75,000 sats

💰 Alby's team & community support

---

## 🔗 Links:
- 🔗[Bitcoin Connect landing page](https://bitcoin-connect.com) 
- 🔗 [GitHub Issue #182](https://github.com/getAlby/bitcoin-connect/issues/182)
- 🔗[](https://usability.yale.edu/web-accessibility/articles/focus-keyboard-operability)
- 🔗 **#bitcoin-connect** [channel on NWC's Discord connecting developers](https://discord.nwc.dev)
- 📧 Contact: [hello@getalby.com](mailto:hello@getalby.com)

---

## 💡 Please Remember:

- 💡 Let the world know if you start working on this issue
- 💡 Work on a feature branch or a forked repository
- 💡 Submit a pull request with a detailed description of your changes
- 💡 Follow basic coding standards and leave helpful comments
- 💡 Test the modal widget's keyboard accessibility across multiple browsers and devices

---

 ♿ #ACCESSIBILITY #OPENNESS #DECENTRALIZATION

💙 Thank you for your contributions and making Bitcoin more accessible and user-friendly for everyone!

**Comment by harith-hacky03** [comment]
I am ready to fix it. Can you assign me please?

**Comment by rolznz** [comment]
@harith-hacky03 normally I do not assign issues to external contributors because there is no guarantee they will do it.

If you are interested, could you make a pull request?

**Comment by chebizarro** [comment]
Hey there, I've made an initial PR (#259) with mostly minor changes which enable both keyboard navigation and screen reader feedback. There's still some testing to be done and a few niggling issues with default buttons, but I'd appreciate if you could give me some feedback if I'm on the right path.

**Comment by Nomishka** [comment]
Hello everyone! @rolznz Is this issue still open or @chebizarro / @harith-hacky03 anyone of you are still working on this?

**Comment by krrish-sehgal** [comment]
Thanks @rolznz , This is the address: bc1qlf0dnf683hj0xh95d77dsjjncs3qadeau87gyp

**Comment by rolznz** [comment]
Bounty paid, thanks @krrish-sehgal !
I'll close this issue and make a follow-up for the final task.
